src/components/aktivitetskrav/historikk/AktivitetskravHistorikk.tsx

```diff
@@ -100,23 +102,21 @@ const HistorikkElement = ({ vurdering }: HistorikkElementProps) => {
   const arsak = vurdering.arsaker[0];
 
   return (
-    <Accordion>
-      <Accordion.Item>
-        <Accordion.Header>{header}</Accordion.Header>
-        <Accordion.Content>
-          {!!arsak && (
-            <Paragraph title={texts.arsakTitle} body={getArsakText(arsak)} />
-          )}
-          {!!vurdering.beskrivelse && (
-            <Paragraph
-              title={texts.beskrivelseTitle}
-              body={vurdering.beskrivelse}
-            />
-          )}
-          {veilederinfo?.navn}
-        </Accordion.Content>
-      </Accordion.Item>
-    </Accordion>
+    <Accordion.Item>
```

Comment:
Tror det skal bli riktigere både styling-messig og semantisk å bruke én Accordion med ett AccordionItem per historikk-element. Samme gjelder for SamtaleAccordion under.